route/car-route.js

```diff
@@ -32,4 +32,16 @@ module.exports = function (router) {
       response.sendText(res, 400, 'bad request');
     }
   });
+  router.delete('/api/car', function (req, res) {
+    try {
+      var car = null;
```

Comment:
the 'var car' is never get reassigned; 
delete api will be called with a query parameter 'id' and other params will not be provided